test/rubygems/test_require.rb

```diff
@@ -422,9 +414,11 @@ def test_require_bundler_with_bundler_version
   end
 
   def silence_warnings
-    old_verbose, $VERBOSE = $VERBOSE, false
-    yield
-  ensure
-    $VERBOSE = old_verbose
+    begin
```

Comment:
the `begin` here shouldn't be necessary

Reply:
It's my mistake for syntax fixes. see  https://github.com/rubygems/rubygems/pull/2141/commits/abc00f1ac9fef0b3d0bc552bd1710561685f9621